client/src/components/Form/index.js

```diff
@@ -29,79 +33,93 @@ class MainForm extends Component {
     });
   };
 
+  handleChange= ({target:{name,value}})=> {
```

Comment:
Why is this handle change here? The point of Formik is to remove the need to write these yourselves.